packages/core/src/Stores/ui-store.js

```diff
@@ -227,6 +227,30 @@ export default class UIStore extends BaseStore {
         return !!this.account_switcher_disabled_message;
     }
 
+    @action.bound
+    filterNotificationMessages() {
+        const new_app_routing_history = this.root_store.common.app_routing_history.slice();
+        const current_platform = getPlatformHeader(new_app_routing_history);
```

Comment:
This won't work if we're redirecting from e.g. DBot to Reports. We'll still maintain the DBot header, but we'd technically be outside the DBot platform. Instead we could we set up a helper in `deriv-app/packages/shared/src/utils/platform/platform.js` that gets the current platform name based on the current route, we can use `isBot` and `isMT5` helpers for this.